File: jvm/ship-conquest/src/main/kotlin/com/example/shipconquest/domain/lobby/Lobby.kt
package com.example.shipconquest.domain.lobby

/**
 * The [Lobby] data class holds the data for the generic information
 * óf a lobby.
 */
data class Lobby(
    val tag: String,
    val name: String,
    val uid: String,
    val username: String,
    val creationTime: Long
)

File: flutter/lib/domain/lobby/lobby.dart
/// Class holding a lobby's most simple information such as [tag], [name],
/// [uid], [username] and [creationTime].
class Lobby {
  final String tag;
  final String name;
  final String uid;
  final String username;
  final int creationTime;

  Lobby(
      {required this.tag,
      required this.name,
      required this.uid,
      required this.username,
      required this.creationTime});
}
